test/Syntax/Parser/tree.swift.result

```diff
@@ -1,8 +1,8 @@
-<s118><s163><s92><s13><NULL/><NULL/><t6>// REQUIRES: syntax_parser_lib
+<s119><s164><s93><s13><NULL/><NULL/><t6>// REQUIRES: syntax_parser_lib
```

Comment:
I’m not sure why changing this test to pass was necessary. I wonder if changing this test is breaking.